praw/objector.py

```diff
@@ -1,10 +1,29 @@
 """Provides the Objector class."""
+import re
+
 from .exceptions import APIException
 
 
 class Objector(object):
     """The objector builds :class:`.RedditBase` objects."""
 
+    @staticmethod
+    def camel_to_snake(name):
+        """Convert camelCase string to snake_case."""
+        first_break_replaced = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
+        return re.sub(
+            '([a-z0-9])([A-Z])', r'\1_\2', first_break_replaced).lower()
+
+    @staticmethod
+    def snake_case_keys(dictionary):
+        """Return a copy of d with keys replaced according to corrections.
```

Comment:
`d` looks like it should be `dictionary`, and it appears there are no longer corrections. (Params should be updated to reflect that as well).